ui/pages/confirm-send-ether/__snapshots__/confirm-send-ether.test.js.snap

```diff
@@ -606,12 +607,13 @@ exports[`ConfirmSendEther should render correct information for for confirm send
                 class="transaction-detail-item__row"
               >
                 <h6
-                  class="box box--margin-top-1 box--margin-bottom-1 box--flex-direction-row typography typography--h7 typography--weight-normal typography--style-normal typography--color-text-alternative"
+                  class="mm-box mm-text mm-text--body-sm mm-box--color-text-alternative"
                 >
                   Amount + gas fee
                 </h6>
                 <h6
-                  class="box box--margin-top-1 box--margin-bottom-1 box--flex-direction-row typography transaction-detail-item__row-subText typography--h7 typography--weight-normal typography--style-normal typography--align-end typography--color-text-alternative"
+                  align="end"
```

Comment:
This looks like a prop being passed to the HTML will need to remove this or adjust it to the correct prop